spec/service_worker/index.bs

```diff
@@ -1784,14 +1790,22 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/
                         <li>Let <var>potentialResponse</var> be a copy of <var>response</var>'s associated <a href="https://fetch.spec.whatwg.org/#concept-response-response">response</a>, except for its body.</li>
                         <li>If <var>response</var>'s body is non-null, run these substeps:
```

Comment:
While here it might be good to make "body" link to https://fetch.spec.whatwg.org/#concept-response-body